cmd/find-junit5-prs/main.go

```
@@ -0,0 +1,537 @@
+}
+
+// generateCandidateURLsFile creates a text file with PR URLs for easy addition to junit5_pr_urls.txt
+func generateCandidateURLsFile(prs []JUnit5PR, outputFile string) {
+	file, err := os.Create(outputFile)
+	if err != nil {
+		fmt.Printf("Error creating file: %v\n", err)
+		return
+	}
+	defer file.Close()
+
+	file.WriteString(fmt.Sprintf("# JUnit 5 migration PR candidates found on %s\n", time.Now().Format("2006-01-02 15:04:05")))
```

golangci-lint (1.64.8)

Error return value of `file.WriteString` is not checked

(errcheck)